extensions/skyportal/static/js/components/SidebarAndHeader.jsx

```diff
@@ -0,0 +1,186 @@
+import React from "react";
```

Comment:
This should not be included here.

Reply:
Well, I feel the same way, but, omitting the colors, where/how do we define Fritz-specific links in the sidebar - see the snapshot [above](https://github.com/fritz-marshal/fritz/pull/11#issuecomment-659217519)? Sources and Groups should probably go to SP, but Alerts are F-specific.

Reply:
The JS components that appear in the sidebar are supposed to be configurable in the config file.  I guess I thought we had already made that change!  If not, let me do that for you now.

Reply:
Yes, please :)